### DISCUSSION TITLE
output infos for inspect and validate as json

### DISCUSSION THREAD

**Discussion by arpu** [donmccurdy/glTF-Transform#1506] [Ideas]
hi,

at the moment there is --format md pretty csv, would it be possible to add json with the cli?

**Comment by donmccurdy** [comment]
Hi! Previously requested in https://github.com/donmccurdy/glTF-Transform/issues/231, with some details there about implementation. I'm open to this as a feature request, and PRs would be welcome, though it isn't something I'm planning to implement myself right now.

**Reply by arpu** [reply]
Thanks for the info!
